app/models/repo.rb

```diff
@@ -0,0 +1,22 @@
+class Repo
+  def self.github
+    Github.new
+  end
+
+  def self.mine(user)
+    data = github.repos.list user: "#{user.screen_name}"
+    get_params(data) || []
+  end
+
+  def self.starred(user)
+    data = github.activity.starring.starred user: "#{user.screen_name}"
+    get_params(data) || []
+  end
+
+  private
```

Comment:
Useless `private` access modifier.<br>Keep a blank line before and after `private`.